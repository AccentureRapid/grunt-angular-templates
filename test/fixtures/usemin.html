<!DOCTYPE html>
    <head>
        <link rel="stylesheet" href="styles/main.css">
        <script src="scripts/vendor/modernizr.min.js"></script>
    </head>
    <body>

    <!-- build:js usemin/foo.js -->
    <script src="usemin/foo.js"></script>
    <!-- endbuild -->

    <!-- build:css usemin/bar.css -->
    <script src="usemin/bar.css"></script>
    <!-- endbuild -->
</body>
</html>
