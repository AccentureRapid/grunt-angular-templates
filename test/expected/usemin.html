<!DOCTYPE html>
    <head>
        <link rel="stylesheet" href="styles/main.css">
        <script src="scripts/vendor/modernizr.min.js"></script>
    </head>
    <body>

    <script src="usemin/foo.js"></script>

    <link rel="stylesheet" href="usemin/bar.css"/>
</body>
</html>
